src/SFA.DAS.QnA.Database/projects/roatp/sections/section2.1.json

```diff
@@ -42,9 +28,45 @@
             "Visible": true,
             "Feedback": null,
             "HasFeedback": false,
-            "NotRequiredConditions": [],
+            "NotRequiredConditions": [
+                {
+                    "Field": "Has-ParentCompany",
+                    "IsOneOf": [
+                        "Yes"
+                    ]
+                }
+            ],
             "ActivatedByPageId": null,
-            "BodyText": ""
+            "BodyText": "<p class='govuk-body'>This section is to ensure your organisation's financial performance and ability to meet ongoing financial commitments meet ESFA's expections.</p><p class='govuk-body'>You may need to provider your organisation's:</p><ul class='govuk-list govuk-list--bullet'><li>financial figures</li><li>full financial statements or management accounts</li></ul><p class='govuk-body'>Based on your answers and uploads, your organisation will receive a financial health grade.</p><p class='govuk-body'>Find out more in <a href='https://www.gov.uk/government/publications/esfa-financial-health-assessment' target='_blank'>ESFA's financial health policy (opens in a new window or tab)</a>.</p><div class='govuk-warning-text'> <span class='govuk-warning-text__icon' aria-hidden='true'>!</span> <strong class='govuk-warning-text__text'> <span class='govuk-warning-text__assistive'>Warning</span>Financial uploads must not be abbreviated, filleted or for a micro entity.</strong> </div>"
+        },
+        {
+            "PageId": "2010",
+            "SequenceId": null,
+            "SectionId": null,
+            "Title": "Financial health assessment",
+            "LinkTitle": "Financial health assessment",
+            "InfoText": "",
+            "Questions": [],
+            "PageOfAnswers": [],
+            "Next": [],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": true,
+            "NotRequiredConditions": [
+                {
+                    "Field": "Has-ParentCompany",
+                    "IsOneOf": [
+                        "No",
+                        ""
```

Comment:
Would it ever be null  (i.e. not defined in ApplicationData) ?

Reply:
I'll add it in - good shout